bouncer/scripts/test/redirect-test.js

```diff
@@ -13,6 +13,7 @@ describe('redirect()', function () {
         })
       };
     };
+    window.console.error = sinon.stub();
```

Comment:
When stubbing existing methods, it is usually better to use `sinon.stub(object, methodName)` and then use `object.method.restore()` afterwards to tear down the stub. Otherwise you can end up overwriting other customizations to the functions that are performed by the test harness.